packages/driver/cypress/integration/issues/5681_spec.js

```diff
@@ -0,0 +1,24 @@
+describe('5681', () => {
+  let error = new ReferenceError('notExistingFunction is not defined')
+
+  before((done) => {
+    cy.on('fail', (err) => {
+      expect(err.message).to.eql(error.message)
+      expect(err.name).to.eql(error.name)
+
+      done()
+    })
+
+    // generate a syntax error
+    // eslint-disable-next-line
+    notExistingFunction()
+  })
+
+  after(() => {
+    cy.get('.action-email').type('fake@email.com')
+  })
+
+  it('green test', () => {
+    cy.visit('https://example.cypress.io/commands/actions')
```

Comment:
Also avoid calling cy.visit to externally hosted site. If you need some DOM to test against, you can visit a local HTML fixture (like `cy.visit('/fixtures/dom.html')`) or create a new HTML fixture to visit and test against depending on what you want.

I don't think visiting any HTML / cypress commands are necessary here, can probably just use standard `expect(true).to.be.true`, `expect(true).to.be.false`, etc.